getting_started/step_by_step/scripting_player_input.rst

```diff
@@ -116,6 +138,39 @@ Here is the complete ``Sprite2D.gd`` file for reference.
 
         position += velocity * delta
 
+ .. code-tab:: csharp C#
+
+    using Godot;
+    
+    public class Sprite : Godot.Sprite
+    {
+        private float Speed = 400;
+        private float AngularSpeed = Mathf.Pi;
```

Comment:
The C# naming convention for fields is camelCase.

```suggestion
        private float speed = 400;
        private float angularSpeed = Mathf.Pi;
```

In the engine we also prefix private fields with an underscore `_` but I'm not sure if that also applies to the documentation.

Reply:
Yes, I'm aware of C# naming conventions. I just tried to match it with the existing documentation. For example, it was merged recently: https://docs.godotengine.org/en/stable/getting_started/step_by_step/scripting_first_script.html
Related PR: https://github.com/godotengine/godot-docs/pull/5991

I thought if it was reviewed and approved two days ago, it would be safe to follow the same rules. 

I'm for using `_` with private variables.

Reply:
Updated with a lower case letter.

Reply:
Ah I missed that PR, it's ok match the existing documentation, the naming can be updated in a separate PR but since you already updated it.

Reply:
So, shall I leave it with the lower case letter or revert back to the upperCase? I think it's better to revert it back.

Reply:
Sure, either way is fine.

Reply:
Reverted back, to match with the current docs.